### ISSUE TITLE
Network calls not showing in the Chrome Dev Tools Network tab

### ISSUE THREAD

**Issue by shawndewet** [EddyVerbruggen/nativescript-plugin-firebase#591]
I have built a Nativescript app based on the Groceries template from the Nativescript documentation [here](https://docs.nativescript.org/angular/tutorial/ng-chapter-2).
I got it up and running and working, and when I call tns debug android I can see all network calls to the Everlive api in the Network tab of Chrome Dev Tools.
I am now embarking on replacing that default implementation with this Firebase plugin, and I can get the list working, I can add groceries and display the list of groceries.
However, none of the calls to the firebase api show up in the Network tab of the Chrome Dev Tools.

Please advise why that would be?  Am I missing something, or could it be an issue with this Firebase Plugin?

**Comment by EddyVerbruggen** [comment]
I'm not sure how those network calls are rerouted to Chrome dev tools in the first place, but I'm guessing there's some listener on NativeScript's http module to wire it up.

The calls to the Firebase backend run outside NativeScripts 'knowledge' through the Firebase SDK using whatever protocol/library Google fancies to use. So it doesn't suprise me you're not seeing those.

Would be nice to see my assumptions verified by someone who has more knowledge of the Dev tools integration.. so perhaps you should open an issue in the NativeScript repo for that.

**Comment by shawndewet** [comment]
Yeah I found the following in their documentation, which (may be of interest/may make sense) to you as the plugin author:
https://docs.nativescript.org/tooling/chrome-devtools#plugin-authors-guide

**Comment by EddyVerbruggen** [comment]
Thanks, I didn't see that before. So the plugin needs to explicitly call those debugging functions when sending/receiving from Firebase. Should be possible, but it's a bit of a hack imo. Still, could be useful to some, so reopening (but not really planning working on it yet).

**Comment by BasyaLipman** [comment]
@shawndewet what are you doing in the meantime? Are you using a different debug tool? I've come across this issue as well - network calls are not showing in the Network tab with my NS+Angular app.

Thing is @EddyVerbruggen , the network calls used to appear nicely in the Network tab. I'm not sure if I stopped seeing them when I moved from debugging Android to debugging iOS or if it's unrelated.

Thanks!

**Comment by rodrigograca31** [comment]
I'm running an ionic/angular/typescript project and I can't see the network calls either.... I love the answer but I need to see the call itself....
using `@ionic-native/http` post method.

**Comment by BasyaLipman** [comment]
@EddyVerbruggen any update here? Or suggestions for the meantime?

**Comment by saipradeepvutukuru** [comment]
Hi, @rodrigograca31 did you find a solution on how to see network calls with @ionic-native/http. Your help is appreciated

**Comment by rodrigograca31** [comment]
@saipradeepvutukuru no.

Maybe setup a proxy or something similar?

**Comment by belkashehaj** [comment]
HI @saipradeepvutukuru, did you find the solution to check network calls?
